README.md

```diff
@@ -14,25 +14,37 @@
 ## Benchmark:
 
 ```
-BenchmarkSort_fnv_10-8                           5000000               365 ns/op             224 B/op          3 allocs/op
-BenchmarkSort_fnv_100-8                           300000              5261 ns/op            1856 B/op          3 allocs/op
-BenchmarkSort_fnv_1000-8                           10000            119462 ns/op           16448 B/op          3 allocs/op
-BenchmarkSortByIndex_fnv_10-8                    3000000               546 ns/op             384 B/op          7 allocs/op
-BenchmarkSortByIndex_fnv_100-8                    200000              5965 ns/op            2928 B/op          7 allocs/op
-BenchmarkSortByIndex_fnv_1000-8                    10000            127732 ns/op           25728 B/op          7 allocs/op
-BenchmarkSortByValue_fnv_10-8                    2000000               962 ns/op             544 B/op         17 allocs/op
-BenchmarkSortByValue_fnv_100-8                    200000              9604 ns/op            4528 B/op        107 allocs/op
-BenchmarkSortByValue_fnv_1000-8                    10000            111741 ns/op           41728 B/op       1007 allocs/op
+BBenchmarkSort_fnv_10-8                                           4812801               240.9 ns/op           216 B/op          4 allocs/op
```

Comment:
`BB`?

Reply:
Frankly, I would rather delete this from README -- raw numbers, without any explanation or meaningful implications. Or, maybe at least add CPU model/RAM capacity. Like in this diff: the numbers have changed even though for most of the benchmarks the code is the same.

Not in this PR, of course. cc @alexvanin